drivers/storage/portworx/util/util.go

```diff
@@ -435,9 +437,16 @@ func StartPort(cluster *corev1.StorageCluster) int {
 
 // KubeletPath returns the kubelet path
 func KubeletPath(cluster *corev1.StorageCluster) string {
+	for _, env := range cluster.Spec.Env {
```

Comment:
Could you add some unit tests to validate CSI deployment/statefulset and PX pod take the env variable?
CSI test example: https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/components_test.go#L5244

Portworx pod test example:
https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/deployment_test.go#L99

Reply:
added unit test in both deployment and components